jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/TabsTests.java

```diff
@@ -0,0 +1,357 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.text.SimpleDateFormat;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+import static com.epam.jdi.light.elements.composite.WebPage.refresh;
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.tabsSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class TabsTests extends TestsInit {
+
+    private final String CLASS_ATTR = "class";
+    private final String ACCENT = "accent";
+    private final String PRIMARY = "primary";
+    private final String DYNAMIC_CONTENT = "Content %s";
+    private final String TAB_GROUP_DEFAULT_CLASS = "mat-tab-group mat-primary";
+    private final List<String> TITLES_DEFAULT_LIST = Arrays.asList("First", "Second", "Third");
+
+    @BeforeMethod
```

Comment:
Are you sure that @BeforeMethod will run before all methods even if the previous test failed?
In my opinion it is necessary use attribute "alwaysRun" with value "true".